src/components/modals/EditDialogModal.tsx

```diff
@@ -237,6 +247,24 @@ class EditDialogModal extends React.Component<Props, ComponentState> {
         })
     }
 
+    @OF.autobind
+    onSaveConflictSave() {
+        // Save the dialog
+        this.onClickSave()
+        this.setState({
+            isSaveConflictModalOpen: false
+        })
+    }
+
+    @OF.autobind
+    onSaveConflictCancel() {
+        // Increment webchat key to reset and clear last input
+        this.setState({
+            isSaveConflictModalOpen: false,
+            webchatKey: this.state.webchatKey + 1
```

Comment:
Interested why changing the key removes the last input.

Reply:
It forces a redraw of webchat from the the train dialog (which hasn't yet been updated)

Reply:
Yea, I know changing key forces remount but I didn't understand how the item got there is we didn't put it there.

I didn't think we gave the web chat a dialog only gave it `history` which was a list of activities. Is that correct?
So it has prop of `history` and internal state of activities and by forcing refresh it will re-use activities from the `history` prop but reset internal state?

Reply:
When the user types in a new input, the webchat display updates to show the new input, but the train dialog doesn't update as we detect the conflict.  The history doesn't have the new input